photon-ml/src/main/scala/com/linkedin/photon/ml/util/DefaultIndexMap.scala

```diff
@@ -20,10 +20,16 @@ package com.linkedin.photon.ml.util
   *
   * @author yizhou
   */
-class DefaultIndexMap(@transient val featureNameToIdMap: Map[String, Int]) extends IndexMap {
+class DefaultIndexMap(@transient private val featureNameToIdMap: Map[String, Int]) extends IndexMap {
```

Comment:
You're right @XianXing, I fixed it.

Generally it's best to keep members private by default, unless there's a really good reason to expose them, because otherwise you give up control over who can change the internal state of the object. In this case it doesn't really matter, since the member is immutable, but it's a good general rule to follow IMO.